src/sentry/web/frontend/accounts.py

```diff
@@ -97,6 +97,7 @@ def recover_confirm(request, user_id, hash):
             form = ChangePasswordRecoverForm(request.POST)
             if form.is_valid():
```

Comment:
I didn't see anywhere that the nonce would get set (or reset) for the active session here. `login_user` is `django.contrib.auth.login` — not `sentry.utils.auth.login` — but I might be overlooking something?